### DISCUSSION TITLE
WW9 - Yaackov

### DISCUSSION THREAD

**Discussion by jacobavitan837** [FPGA-MAFIA/fpga_mafia#512] [Weekly]
# Completed:
- #480
- #483
- #507 (probably I guess)
# WIP:
- #482
# Not Started
- #481
- #503

**Comment by amichai-bd** [comment]
- [x] Please update your weekly to represent all your task! (no only the 3)
![image](https://github.com/FPGA-MAFIA/fpga_mafia/assets/81047407/86e7a313-118f-47a9-bc56-c1fcc9b617ff)
- [x] add another bullet for "Not Started"

**Comment by amichai-bd** [comment]
Also update the category to be under "weekly"

**Reply by jacobavitan837** [reply]
I hope to receive more feedbacks on this discussion @amichai-bd
and thanks for the help

**Comment by amichai-bd** [comment]
@jacobavitan837 
You haven't moved this to the correct category.
Please make sure it's under "weekly"
"
![image](https://github.com/FPGA-MAFIA/fpga_mafia/assets/81047407/f3f8548d-4027-4998-8424-38f4377def5c)
"

**Reply by jacobavitan837** [reply]
I cant change the category to weekly

<img width="413" alt="image" src="https://github.com/FPGA-MAFIA/fpga_mafia/assets/160140405/23e038b5-fea6-4f21-a007-2688192b8f10">

**Reply by amichai-bd** [reply]
please try now
